src/networkmessage.h

```diff
@@ -126,16 +126,16 @@ class NetworkMessage
 		}
 
 		uint8_t* getBuffer() {
-			return buffer;
+			return buffer.begin();
 		}
 
 		const uint8_t* getBuffer() const {
-			return buffer;
+			return buffer.begin();
```

Comment:
don't know if you noticed, but this doesn't compile

Reply:
It does compile and I'm using the server with this patch

Reply:
hmm, windows-cl.exe build fails: https://github.com/otland/forgottenserver/runs/5705141976?check_suite_focus=true

Reply:
Aha, MS' shitty implementation for array does not decay iterator to pointer. I wonder how they managed to do that